models/actions/runner_token.go

```diff
@@ -22,7 +22,7 @@ type ActionRunnerToken struct {
 	Owner    *user_model.User       `xorm:"-"`
 	RepoID   int64                  `xorm:"index"` // repo level runner, if orgid also is zero, then it's a global
 	Repo     *repo_model.Repository `xorm:"-"`
-	IsActive bool
+	IsActive bool                   // true means it can be used
```

Comment:
Should we include a migration to reverse the value in DB?

Reply:
Oh, never mind. I noticed the update operation in ﻿NewRunnerToken that deactivates old tokens.